pkg/util/timeutil/lowercase/main.go

```diff
@@ -47,11 +47,11 @@ var lowercaseTimezones = map[string]string{
 `
 
 func main() {
+	var filenameFlag = flag.String("filename", "lowercase_timezones.go", "filename path")
 	flag.Parse()
-
-	const fileName = "lowercase_timezones.go"
-
+	fileName := *filenameFlag
 	zipdataFile, err := os.Open(filepath.Join(runtime.GOROOT(), "lib", "time", "zoneinfo.zip"))
```

Comment:
This seems wrong. We shouldn't ever be inferring the location of anything in the Bazel sandbox like this, we should compute it directly and then pass it in as a flag.

Reply:
Sorry if I am getting incorrectly, but we are passing the `GOROOT` and the `zoneinfo.zip` right on the `tools` of the `genrule`

Reply:
DON'T pass `GOROOT` in from Bazel. `GOROOT` doesn't exist, as far as Bazel is concerned.

1. Depend on `"@go_sdk//:lib/time/zoneinfo.zip",` in the `genrule`.
2. In the `genrule`, get `$(location @go_sdk//:lib/time/zoneinfo.zip)`, and pass it down to this as an argument.
3. Use that to find where `zoneinfo.zip` is located.

This is how you should always find files you depend on in Bazel. It is NEVER a good idea to find a random directory and try to derive paths from the top directory.